FMC increased leniency (#34)
CarterKoala/wca-regulations-wrc: wca-guidelines.md

- E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
- E2c3+) [CLARIFICATION] If a move is not clearly blacked out/scribbled out and it is not clear whether the move is part of the solution, the judge should consider it an ambiguous move sequence (see [Regulation E2c2](regulations:regulation:E2c2) and [Guideline E2c2+](guidelines:guideline:E2c2+)).

> - E2c4+) [EXAMPLE] Examples of interpreting symbols or combinations of symbols that are not defined in [Regulation 12a](regulations:regulation:12a): [x] should be interpret as x. (R * U', * = F2) should be interpreted as R U' F2. (R U)' should be interpreted as R U'. R++ should be interpreted as R. R3' is not permitted.

Reviewer: ```suggestion
- E2c4+) [EXAMPLE] Examples of interpreting symbols or combinations of symbols that are not defined in [Regulation 12a](regulations:regulation:12a): [x] should be interpreted as x. (R * U', * = F2) should be interpreted as R U' F2. (R U)' should be interpreted as R U'. R++ should be interpreted as R. R3' is not permitted.
```
Fix typo

> - E2c4++) [ADDITION] Moves such as R2' should be interpreted as R2, whereas otherwise they would have resulted in a disqualification of the attempt (DNF).

Reviewer: ```suggestion
- E2c4++) [ADDITION] Moves with both an apostrophe and a 2 should be interpreted as if there were no apostrophe (e.g. R2' should be interpreted as R2).
```
Reword, otherwise it reads like L2' should be interpreted as R2. Remove redundant mention of DNF since that's already in the base regulation.

- E2b++) [ADDITION] A competitor may join a 3x3x3 Fewest Moves attempt that is already in progress, at the discretion of the WCA Delegate. In this case, their time limit is the time remaining (i.e. they must finish when other competitors reach their 60 minute time limit). Exception: a competitor must not be permitted to join a 3x3x3 Fewest Moves attempt if any competitor has finished their attempt already (see [Guideline E2b+](guidelines:guideline:E2b+)), or if there is any other risk that information about the scramble may have become known to the competitor who wishes to join.
- E2c+) [CLARIFICATION] Although the judge may provide a standard sheet for competitors to submit solutions, a competitor may submit a solution on a different piece of paper. (Note that the piece of paper must come from the judge, according to [Regulation E3a](regulations:regulation:E3a).)
- E2c1+) [EXPLANATION] A competition registrant ID is an identifier (e.g. a number) assigned to a competitor for the competition. Some competitions use this to keep track of score sheets and enter results.

> - E2c1++) [ADDITION] Information to identify the attempt may be added by the competitor after the attempt has been submitted but before the end of the competition, at the discretion of the judge and under their direct supervision. The competitor must not write anything after the time limit without the judge's supervision and must not make any changes to the solution. Penalty: disqualification of the attempt (DNF).

Reviewer 2: This should probably be moved into E2c1 regardless of whether or not we implement this part of it

- E2c2++) [REMINDER] In the past, bracket notations (e.g. [r] or [u2]) were permitted for Fewest Moves. Only rotations based on x, y, or z are permitted now (see [Regulation 12a4a](regulations:regulation:12a4a)).
- E2c3+) [CLARIFICATION] If a move is not clearly blacked out/scribbled out and it is not clear whether the move is part of the solution, the judge should consider it an ambiguous move sequence (see [Regulation E2c2](regulations:regulation:E2c2) and [Guideline E2c2+](guidelines:guideline:E2c2+)).

> - E2c4+) [EXAMPLE] Examples of interpreting symbols or combinations of symbols that are not defined in [Regulation 12a](regulations:regulation:12a): [x] should be interpret as x. (R * U', * = F2) should be interpreted as R U' F2. (R U)' should be interpreted as R U'. R++ should be interpreted as R. R3' is not permitted.

Reviewer 3: I'm a bit worried that this is going too far, as it appears to define a way to parse/interpret algorithms that is inconsistent with conventions. 😕

Reviewer 4: Is there a specific example you are concerned about? I agree that competitors writing something like `[R U R', D2]` likely didn't want the move sequence to be parsed as `R U R' D2`, but we can either DNF the attempt directly, or we can define a way to transform whatever the competitor has written into an algorithm that can be checked against the scramble.

- E2c4++) [ADDITION] Moves such as R2' should be interpreted as R2, whereas otherwise they would have resulted in a disqualification of the attempt (DNF).

> - E2c5+) [EXAMPLE] Y should be interpreted as y. UW, uw, and uW should be interpreted as Uw. r should be interpreted as R, not Rw.

Reviewer 3: I think `r` is substantially different from `R` compared to the other cases. The others can be considered slightly inconsistent writing size, but `R` is clearly spelled out as a move in the Regs and on the FMC sheet and `r` is not.

Author: I think we need to consider *why* there is a need to disqualify competitors who have mistakenly written r instead of R. It shouldn’t need to make a difference to the solution. 
There are a lot of young and/or inexperienced cubers who want to do FMC now but rulings like this are so unnecessarily strict that many of them get DNF’d and don’t want to compete in it again. Ultimately that’s what this set of proposed changes is trying to fix.

Reviewer 4: There are already situations where it isn't clear if a character is capitalized (U/u, X/x, etc.), and DNFing those seems unnecessarily strict. Even in cases where the difference between lower-case and upper-case characters is clear there is no intrinsic reason why an `r` should lead to a DNF immediately if we can define that we'll unambiguously resolve lower-case characters to their corresponding upper-case ones.

Reviewer 5: Instead of giving examples, what if we say that solutions are case-insensitive?

> - E2c5+) [EXAMPLE] Y should be interpreted as y. UW, uw, and uW should be interpreted as Uw. r should be interpreted as R, not Rw.

Reviewer 2: ```suggestion
- E2c6+) [EXAMPLE] Y should be interpreted as y. UW, uw, and uW should be interpreted as Uw. r should be interpreted as R, not Rw.
```

> - E2c7+) [ADDITION] Competitors must not intentionally make their solution harder to judge by using external symbols, incorrect capitalization, or by any other means. Penalty: disqualification of the attempt (DNF) at the discretion of the WCA Delegate.

Reviewer 2: Remove this if added to E2c8

> - E2c7+) [ADDITION] Competitors must not intentionally make their solution harder to judge with excessive use of illegal symbols, incorrect capitalization, or by other means, at the discretion of the WCA Delegate. Penalty: disqualification of the attempt (DNF)

Reviewer 2: This should be in the regs rather than the guidelines

Reviewer 3: Also, small drive-by nit: we don't use "illegal" in the Regs, "non-permitted" matches other Regs and Guidelines.

> - E2c7+) [ADDITION] Competitors must not intentionally make their solution harder to judge with excessive use of illegal symbols, incorrect capitalization, or by other means, at the discretion of the WCA Delegate. Penalty: disqualification of the attempt (DNF)

Reviewer: ```suggestion
- E2c7+) [ADDITION] Competitors must not intentionally make their solution harder to judge by using illegal symbols, incorrect capitalization, or by any other means. Penalty: disqualification of the attempt (DNF) at the discretion of the WCA Delegate. 
```

Reviewer 2: ```suggestion
- E2c7+) [ADDITION] Competitors must not intentionally make their solution harder to judge by using external symbols, incorrect capitalization, or by any other means. Penalty: disqualification of the attempt (DNF) at the discretion of the WCA Delegate.
```
